docs/getting-started-guides/rkt/index.md

```diff
@@ -14,7 +14,7 @@ This document describes how to run Kubernetes using [rkt](https://github.com/cor
 
 - The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html) must be running on the node.
 
-### Setup network
+### Setup Network
```

Comment:
Actually, sentence casing is correct for headings. Title Casing Like This is only for page titles.